doc/model/discretization.rst

```diff
@@ -0,0 +1,208 @@
+=======================
+ Navier-Stokes with DG
+=======================
+
+.. _disc-strat:
+
+Discretization Strategy
+=======================
+
+How to discretize the conservation equations with DG, including how to handle the required fluxes,
+particularly in the viscous setting, is a current topic of research and internal discussion.  The
+following references are useful:
+
+* "The DG Book:" Nodal Discontinuous Galerkin Methods, [Hesthaven_2008]_
+* The BR1 algorithm for discretization of Navier-Stokes, [Bassi_1997]_
+* NS with reactions, [Ihme_2014]_, and [Cook_2009]_
+* The BR2 algorithm, [Bassi_2000]_
+* [Ayuso_2009]_
+
+*MIRGE-Com* currently employs a strategy akin to the BR1 algorithm outlined in [Bassi_1997]_, but
+with thermal terms and chemical reaction sources as outlined in [Ihme_2014]_ and [Cook_2009]_.
+
+Navier-Stokes BR1
+=================
+
+The main system of equations we solve in the BR1 approach is summarized as follows:
+
+The compressible NS equations are rewritten as the following coupled system for two unknowns, $\mathbf{Q}$
+and $\mathbf{S}$:
+
+$$
+\mathbf{S} - \nabla{\mathbf{Q}} &= \mathbf{0}\quad \text{aux eqn}\\
+\frac{\partial \mathbf{Q}}{\partial t} + \underbrace{\nabla\cdot\mathbf{F}^I(\mathbf{Q}) - \nabla\cdot\mathbf{F}^V(\mathbf{Q},\mathbf{S})}_{= \nabla\cdot\mathbf{F}(\mathbf{Q}, \mathbf{S})} &= \mathbf{0} \quad \text{primary eqn}
+$$
+
+Let $\Omega_h$ denote a collection of disjoint elements $E$. Then the DG formulation is obtained by multiplying by ''test functions'' $\mathbf{v}_h$, $\mathbf{w}_h$ (one for each equation repsectively) and integrating over each element:
+
+$$
+\sum_{E\in\Omega_h} \left\lbrack \int_E \mathbf{v}_h\cdot\frac{\partial \mathbf{Q}_h}{\partial t} d\Omega + \oint_{\partial E} \mathbf{v}_h\mathbf{h} d\sigma - \int_E \nabla\mathbf{v}_h\cdot\mathbf{F}(\mathbf{Q}_h, \mathbf{S}_h)d\Omega\right\rbrack &= 0, \quad \forall \mathbf{v}_h \\
+\sum_{E\in\Omega_h}\left\lbrack \int_E \mathbf{w}_h\cdot \mathbf{S}_h d\Omega - \oint_{\partial E} \mathbf{w}_h\cdot\mathbf{H}_s d\sigma + \int_E \nabla\mathbf{w}_h\cdot\mathbf{Q}_h d\Omega\right\rbrack &= 0, \quad \forall \mathbf{w}_h,
+$$
+
+where $\mathbf{h} = \mathbf{h}_e - \mathbf{h}_v$ is a **numerical flux** consisting of a **invicid** and **viscous** component respectively, and $\mathbf{H}_s$ is the **gradient numerical flux** for the auxiliary equation.
+
+Since $\mathbf{F}^I(\mathbf{Q}_h)\cdot\mathbf{n}$ is discontinuous, the quantities are allowed to vary on either side of a shared element boundary. That is, $\mathbf{F}^I(\mathbf{Q}^+_h)\cdot\mathbf{n}^+ \neq \mathbf{F}^I(\mathbf{Q}^-_h)\cdot\mathbf{n}^-$, where $\mathbf{n}^\pm$ denotes the ''cell-local'' (or interior) normal with respect to the element $E^\pm$ which share a face: $\partial E^+ \cap \partial E^- \neq \emptyset$. Similarly for $\mathbf{F}^V(\mathbf{Q}_h, \mathbf{S}_h)\cdot\mathbf{n}$ and $\mathbf{Q}_h\mathbf{n}$.
```

Comment:
```suggestion
where $\mathbf{h} = \mathbf{h}_e - \mathbf{h}_v$ is a **numerical flux** consisting of a **invicid** and **viscous** component respectively, and $\mathbf{H}_s$ is the **gradient numerical flux** for the auxiliary equation. Here, we use the subscript "h" to denote discretized quantities.

Since $\mathbf{F}^I(\mathbf{Q}_h)\cdot\mathbf{n}$ is discontinuous, the quantities are allowed to vary on either side of a shared element boundary. That is, $\mathbf{F}^I(\mathbf{Q}^+_h)\cdot\mathbf{n}^+ \neq \mathbf{F}^I(\mathbf{Q}^-_h)\cdot\mathbf{n}^-$, where $\mathbf{n}^\pm$ denotes the ''cell-local'' (or interior) normal with respect to the element $E^\pm$ which share a face: $\partial E^+ \cap \partial E^- \neq \emptyset$. Similarly for $\mathbf{F}^V(\mathbf{Q}_h, \mathbf{\Sigma}_h)\cdot\mathbf{n}$ and $\mathbf{Q}_h\mathbf{n}$.
```